docs/source/index.rst

```diff
@@ -7,9 +7,11 @@ Envisage Documentation
 
     envisage_core_documentation/index
     tasks_user_manual/index
+    changelog
```

Comment:
Bikeshedding, but I'd be tempted to place this after the API docs, given that it is a fairly large chunk of the ToC and I think more people are likely to want to access the API docs than the changelog.

Reply:
Done in 439a1cfacc71d6e877bcd988d21b9243da9ec412